.travis.yml

```diff
@@ -24,6 +24,7 @@ matrix:
         - TEST_MATPLOTLIB="true"
         - TEST_THEANO="true"
         - TEST_ASCII="true"
+        - TEST_COVERAGE="true"
```

Comment:
Can you just report the code coverage during one of the other tests above? It's best to avoid adding new matrix entries, because our tests are already too long.